src/app/modules/group/components/group-manager-list/group-manager-list.component.ts

```diff
@@ -1,28 +1,39 @@
-import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { Component, Input, OnChanges, OnDestroy, SimpleChanges } from '@angular/core';
 import { GetGroupManagersService, Manager } from '../../http-services/get-group-managers.service';
 import { GroupData } from '../../services/group-datasource.service';
 import { RemoveGroupManagerService } from '../../http-services/remove-group-manager.service';
 import { ActionFeedbackService } from '../../../../shared/services/action-feedback.service';
-import { forkJoin } from 'rxjs';
+import { forkJoin, ReplaySubject, Subject } from 'rxjs';
+import { distinct, switchMap, map } from 'rxjs/operators';
+import { mapToFetchState } from '../../../../shared/operators/state';
 
 @Component({
   selector: 'alg-group-manager-list',
   templateUrl: './group-manager-list.component.html',
   styleUrls: [ './group-manager-list.component.scss' ]
 })
-export class GroupManagerListComponent implements OnChanges {
+export class GroupManagerListComponent implements OnChanges, OnDestroy {
 
   @Input() groupData?: GroupData;
 
-  managers: (Manager & { canManageAsText: string })[] = [];
-
-  state: 'loading' | 'ready' | 'error' = 'loading';
-
   selection: Manager[] = [];
   removalInProgress = false;
   isPermissionsEditDialogOpened = false;
   dialogManager?: Manager & { canManageAsText: string };
 
+  private refresh$ = new Subject<void>();
+  private readonly groupId$ = new ReplaySubject<string>(1);
+  readonly state$ = this.groupId$.pipe(
+    distinct(),
```

Comment:
Although the case will probably not happen, we should use `distinctUntilChanged` here:
If the id is `1`, then `2` and then `1` again, we want to refetch item 1

Reply:
Oops, I mixed both as well. I think most of usage of `distinct` in the project should be `distinctUntilChanged`. 
I create an issue...

Reply:
Fixed